org.mwc.cmap.core/src/org/mwc/cmap/core/property_support/TagListHelper.java

```diff
@@ -35,7 +36,21 @@ public TagListHelper(final String[] theTags,
 
 	public CellEditor getCellEditorFor(final Composite parent)
 	{
-		return new ComboBoxCellEditor(parent, _theTags);
+		return new ComboBoxCellEditor(parent, _theTags, SWT.READ_ONLY){

```

Comment:
I made it readonly mode as if not user can enter any text and it throw error when user do that